docker/anet-mssql-linux/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# anet-mssql
+# Maintainers: NCI Agency (VassilIordanov on GitHub)
+# GitRepo: https://github.com/NCI-Agency/anet
+
+FROM ubuntu:16.04
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y apt-transport-https curl  && \
+    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
+    curl https://packages.microsoft.com/config/ubuntu/16.04/mssql-server.list | tee /etc/apt/sources.list.d/mssql-server.list && \
+    curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list | tee /etc/apt/sources.list.d/msprod.list && \
+    apt-get update && \
+    ACCEPT_EULA=Y apt-get install -y mssql-server=14.0.600.250-2 mssql-server-fts=14.0.600.250-2 mssql-tools unixodbc-dev locales && \
```

Comment:
Note that if you really must have ACCEPT_EULA here, the line should probably read:
`export ACCEPT_EULA=Y apt-get install -y mssql-server=14.0.600.250-2 mssql-server-fts=14.0.600.250-2 mssql-tools unixodbc-dev locales && \`
("export " at the start of the line).